starter/share/modules1/format_mod.F90

```diff
@@ -1,45 +1,527 @@
+        CHARACTER(LEN=50) :: FMW_X_I_3F_2I
+        CHARACTER(LEN=50) :: FMW_2I_2F_I_F
+        CHARACTER(LEN=50) :: FMW_I_DA
+        CHARACTER(LEN=50) :: FMW_I_X_3F_I
+        CHARACTER(LEN=50) :: FMW_I_A_4F
+        CHARACTER(LEN=50) :: FMW_2I_F_3I_F
+        CHARACTER(LEN=50) :: FMW_2I_F_2I_X_I
+        CHARACTER(LEN=50) :: FMW_4I_2X_2F
+        CHARACTER(LEN=50) :: FMW_I_2F_I
+        CHARACTER(LEN=50) :: FMW_F_2I_F_X_I_F
+        CHARACTER(LEN=50) :: FMW_2F_I_2F_I
+        CHARACTER(LEN=50) :: FMW_9A_I
+        CHARACTER(LEN=50) :: FMW_2I_F_2I_3A_I
+        CHARACTER(LEN=50) :: FMW_3I_A
+        CHARACTER(LEN=50) :: FMW_2I_F_3I_F_I
+        CHARACTER(LEN=50) :: FMW_2I_F_2I_2A_I
+        CHARACTER(LEN=50) :: FMW_A_2I_X_I
+        CHARACTER(LEN=50) :: FMW_F_3I
+        CHARACTER(LEN=50) :: FMW_F_I_2F_I
+        CHARACTER(LEN=50) :: FMW_I_4F
+        CHARACTER(LEN=50) :: FMW_2I_F_2I_F
+        CHARACTER(LEN=50) :: FMW_2I_F_2I_F_I
+        CHARACTER(LEN=50) :: FMW_I_X_2F_3I
+        CHARACTER(LEN=50) :: FMW_4I_X_I
+        CHARACTER(LEN=50) :: FMW_F_I_X_F_I_F
+        CHARACTER(LEN=50) :: FMW_F_I_X_F_I
+        CHARACTER(LEN=50) :: FMW_2F_I_F_I
+        CHARACTER(LEN=50) :: FMW_F_2I
+        CHARACTER(LEN=50) :: FMW_3F_X_I_F
+        CHARACTER(LEN=50) :: FMW_I_2X_4I
+
+        INTEGER LFIELD
+              
+      contains
+      
+
+! ======================================================================================================================
+!                                                   PROCEDURES
+! ======================================================================================================================
+!! \brief Here is a small description of the routine, [after the header]
+!! \details if needed, more details can be added here
+      SUBROUTINE format_initialization(INVERS,IFIX)
+        IMPLICIT NONE
+! ----------------------------------------------------------------------------------------------------------------------
+!                                                   Arguments
+! ----------------------------------------------------------------------------------------------------------------------
+        INTEGER,INTENT(IN) :: INVERS
+        INTEGER,INTENT(IN) :: IFIX
+! ----------------------------------------------------------------------------------------------------------------------
+!                                                   External functions
+! ----------------------------------------------------------------------------------------------------------------------
+! [ external functions must be kept to mimimum ]
+! ----------------------------------------------------------------------------------------------------------------------
+!                                                   Body
+! ----------------------------------------------------------------------------------------------------------------------
+      IF (INVERS < 50 .OR. IFIX == 2) THEN
```

Comment:
Do we still need to support INVERS < 50 .OR. IFIX == 2?
If no, we could convert the global variables into PARAMETER.

Reply:
Hi  @laurent-altr ,
thank you for reporting this point.
Yes INVERS < 50 is no longer supporter and IFIX argument is always 0. So we can remove this part.
In addition to that I removed all format definitions which were not used.
Format Definitions are now defined with 'PARAMETER' attribute.